node.js

```diff
@@ -400,64 +493,67 @@ async function join(known_node) {
         predecessor = _self;
     }
 
-    // TODO move keys: (predecessor, n]; i.e., (predecessor, _self]
+    // TODO migrate keys: (predecessor, n]; i.e., (predecessor, _self]
+    await migrate_keys();
+
+    // initialize successor table - deviates from SIGCOMM
```

Comment:
What is your intention behind mentioning this deviation? Is this something you think that needs to be fixed? What conclusion do you expect me to make reading this?